src/screens/SurveyDetail/QuestionBuilders/buildQuestionList.js

```diff
@@ -0,0 +1,21 @@
+// Build question list and answers from the api for easier of question and
+// answers across the components
+const buildQuestionList = (details) => {
+  let questions = details.data.relationships.questions.data;
+  const included = details.included;
+
+  questions.forEach((question, index) => {
+    const question_details = included.find(
+      (details) => details.id === question.id
+    );
+
+    questions[index]['type'] = question_details.attributes.display_type;
+    questions[index]['pick'] = question_details.attributes.pick;
+    questions[index]['text'] = question_details.attributes.text;
+    questions[index]['answers'] = question_details.relationships.answers.data;
```

Comment:
Not sure why we're not using `question` here 🤔 

```suggestion
    question['type'] = question_details.attributes.display_type;
    question['pick'] = question_details.attributes.pick;
    question['text'] = question_details.attributes.text;
    question['answers'] = question_details.relationships.answers.data;
```

Reply:
@carryall fixed reviews in [c300c8b](https://github.com/rafayet-monon/surveyor/pull/28/commits/c300c8b52f96bcfbb3a8ebe9eb92f84ee39b2c9e)